### PR TITLE
emit blur event with doc id as string

### PR DESCRIPTION
**Pull request by dheniges**
This was causing an error on document join room, since it wants a string, not a number. Also makes it consistent with the other events, which all seem to be passing strings

### CODE REVIEW

File: web/src/client/js/actions/userSync.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ export const emitJoinDocumentRoom = (documentId) => {
       return UserSync.socketError()
     }
     dispatch(UserSync.emitJoinDocumentRoom(documentId))
+    console.log('emitting joinRoom for ' + documentId)
```
Comment (jjidt) [inline web/src/client/js/actions/userSync.js]: wanna leave these logs in there?

Comment (jjidt) [reply web/src/client/js/actions/userSync.js]: ^

Comment (dheniges) [reply web/src/client/js/actions/userSync.js]: oops no, will update here

**Comment by dheniges** [comment]
@jjidt no more console

**Review by jjidt — APPROVED** [review]

**Pull request merged**